src/components/ApplicationPrompt.astro

```diff
@@ -0,0 +1,72 @@
+---
+import { addDays, DateFormatters } from '~utils';
+export interface Props {
+	cohortKickoffDate: Date;
+}
+
+const { cohortKickoffDate } = Astro.props as Props;
+
+// The application window is open for 1 week,
+// starting 2 weeks before the cohort kickoff
+// and ending 1 week before the cohort kickoff
+// (we're using 15 days to accout for Christmas this quarter)
+const [windowStart, windowEnd] = [
+	addDays(cohortKickoffDate, -15),
+	addDays(cohortKickoffDate, -7),
+];
```

Comment:
@segdeha Is this general logic (and my exception for christmas) correct? If we can't rely on "Kickoff minus 2 weeks" we could have the component accept a kickoff date and a tuple to indicate the application window

Reply:
I think I'd prefer to explicitly pass in dates for the start and end of the application window. That way, we can more easily work around things like holidays without potentially having to change the component.